openpype/hosts/max/plugins/publish/extract_tycache.py

```diff
@@ -0,0 +1,159 @@
+import os
+
+import pyblish.api
+from pymxs import runtime as rt
+
+from openpype.hosts.max.api import maintained_selection
+from openpype.pipeline import publish
+
+
+class ExtractTyCache(publish.Extractor):
+    """
+    Extract tycache format with tyFlow operators.
+    Notes:
```

Comment:
```suggestion
    """Extract tycache format with tyFlow operators.
    
    Notes:
```
I've mostly seen this code style.